wiki/Beatmap_Spotlights/Leaderboards/en.md

```diff
@@ -0,0 +1,272 @@
+---
+tags:
+  - Spotlights
+  - Charts
+  - Ranking Charts
+  - Spotlights leaderboard
+---
+
+# Beatmap Spotlights Test Run (Summer 2020): Ranking
+
+Here you can find the seasonal and weekly lobby leaderboards for each playlist.
+
+## Seasonal TOP 50 Leaderboard
+
+The seasonal leaderboard sums up the weekly results of the timeshift lobbies. You can also find the leaderboard right [here](https://docs.google.com/spreadsheets/d/e/2PACX-1vRF46ZtScaNWb5KLAF0-CwJhTySle5r3xVhg15x1nTOXYrMWNpRx8HbsRizaQ2AwJpKARJRTtyHbdxs/pubhtml#)
+
+### osu!
+
+| Placement | Name | Badge |
+| :--: | :--: | :--: |
+| 1. | [Fangzy](https://osu.ppy.sh/users/4497691) | ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/osu/ri_2.png) |
```

Comment:
an advice that will save you some nerves and time (not to say this is more of a requirement): as you're reusing every badge in this article a couple of times, make the reference-styled image links: https://github.com/ppy/osu-wiki/blob/master/wiki/Article_Styling_Criteria/en.md#links

give them names like `osu-ri-2`, or `mania-platinum-1`

Reply:
Will take a look at this at a later point, at first, the wiki article is not really helpful in explaining how to do that.

Reply:
it's easy: instead of `[title](URL)`, you do:
```markdown
![][eggplant image] [Eggplants][eggplant url] are rich in vitamin K.

...

<!-- (the bottom of the article) -->
[eggplant image]: https://egglpant.com/favicon.png
[eggplant url]: https://eggplants.com
```

Reply:
This should be updated now.